src/index.scss

```diff
@@ -38,11 +38,14 @@ body {
 	height: calc(100vh - #{$header-height});
 	width: 100vw;
 	
+	.red-text {
```

Comment:
I don't want to be That Guy, but... Is there an appropriate Bootstrap CSS class to use instead?  That would allow us to use the framework without adding a class we later remove.  I'm glad to approve this as-is regardless.

Reply:
Oh, the .red-text is a silly class I added just for a short period so that you can see the text I added...this will be removed later.  There might be a bootstrap class for this, but I figured this was simple enough.

Reply:
Indeed it is.  Thank you for doing the heavy lifting to get these pages fleshed out, too.  I'm happy to do the merge.